enclave/crypto/openssl/init.c

```diff
@@ -6,48 +6,48 @@
 #include <openenclave/internal/trace.h>
 #include <openssl/engine.h>
 static oe_once_t _openssl_initialize_once;
-static ENGINE* eng;
+static ENGINE* rdrand_engine;
```

Comment:
```suggestion
static ENGINE* _rdrand_engine;
```
The naming convention is to prefix static variables with `_`. Same for the variable below.